### PR TITLE
opentelemetry-collector-contrib-8267 Correcting 5m and 15m normalized…

### PR DESCRIPTION
**Pull request by shree007**
**Description:** 
wrong 5m and 15m normalized load - using 1m normalized load and normalizing it again

**Link to tracking Issue:** #8267

### CODE REVIEW

**Comment by dmitryax** [comment]
@shree007 Please add a test case and a changelog entry

**Comment by shree007** [comment]
Got it. I will add

File: receiver/hostmetricsreceiver/internal/scraper/loadscraper/load_scraper.go [hunk 1]
```diff
@@ -77,8 +77,8 @@ func (s *scraper) scrape(_ context.Context) (pdata.Metrics, error) {
 	if s.config.CPUAverage {
 		divisor := float64(runtime.NumCPU())
 		avgLoadValues.Load1 = avgLoadValues.Load1 / divisor
-		avgLoadValues.Load5 = avgLoadValues.Load1 / divisor
-		avgLoadValues.Load15 = avgLoadValues.Load1 / divisor
+		avgLoadValues.Load5 = avgLoadValues.Load5 / divisor
```
Comment (anuraaga) [inline receiver/hostmetricsreceiver/internal/scraper/loadscraper/load_scraper.go:80]: Can you add a unit test to verify this change / regressions?

Comment (shree007) [reply receiver/hostmetricsreceiver/internal/scraper/loadscraper/load_scraper.go:80]: sure

**Comment by shree007** [comment]
I will start to work on it from tomorrow.

File: receiver/hostmetricsreceiver/internal/scraper/loadscraper/load_scraper.go (hunk 1 above)
Comment (shree007) [reply receiver/hostmetricsreceiver/internal/scraper/loadscraper/load_scraper.go:80]: @dmitryax  I am seeing that Test case is aleady [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/internal/scraper/loadscraper/load_scraper_test.go#L119)
If there is any specific thing you want me to check, please let me know.

**Comment by shree007** [comment]
@dmitryax  I am seeing that Test case is aleady [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/hostmetricsreceiver/internal/scraper/loadscraper/load_scraper_test.go#L119)
If there is any specific thing you want me to check, please let me know.

**Comment by dmitryax** [comment]
> If there is any specific thing you want me to check, please let me know.

The test must be incorrect if if didn't catch this issue. Please update the test and make sure it's failing without your fix.

**Comment by sodabrew** [comment]
Any news on this PR? Looks like a pretty straightforward fix. The current unit test simply looks that _data exists_ and not that the data is correct. Perhaps the test suite should run for 15 minutes to generate enough data points to test the average?

**Comment by dmitryax** [comment]
@sodabrew help would be appreciated. Feel free to take it if from here if you have a chance

**Comment by shree007** [comment]
@dmitryax @sodabrew I have reverted my changes to validate test case As discussed in this thread. I am keeping an eye on it.
Sorry for inactivity, I was on leave and away from keyboard.

**Comment by sodabrew** [comment]
Looks like this didn't make the 0.51 release.

File: receiver/hostmetricsreceiver/internal/scraper/loadscraper/load_scraper.go (hunk 1 above)
Comment (sodabrew) [reply receiver/hostmetricsreceiver/internal/scraper/loadscraper/load_scraper.go:78]: Respectfully, there is a unit test: it verifies the data is present.

Leaving the code wrong to berate the contributor for not working out how to make a test that calculates a 15 minute average seems like an unnecessary harm.

**Comment by shree007** [comment]
> 
@sodabrew there is multiple test is being failed, I need to look into it. Please feel free to take over if you have bandwidth.

**Comment by sodabrew** [comment]
@shree007 Could you simply rebase this PR to current master? The test failures look unrelated to your change. It would be nice if @codeboten could merge this for 0.52 tomorrow

**Comment by sodabrew** [comment]
The test for this that could even have been updated was just disabled in #10030

**Comment by shree007** [comment]
Sure.

**Comment by sodabrew** [comment]
Got my version landed in #10301! Thanks again for this PR that got it started! Please do try a current build to double-check that it works for you

**Comment by djaglowski** [comment]
Closing this, as it appears to have been addressed by #10301

**Pull request closed without merging**